.npmignore

```diff
@@ -5,4 +5,6 @@
 !README.md
 !package.json
 !yarn.lock
+!formats/utils/color-factory/**/*
```

Comment:
What do you all think about the file organization for this?

Reply:
This directory structure makes sense!